portfolio/src/main/webapp/index.html

```diff
@@ -2,62 +2,111 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>DAVIDVARGAS</title>
+    <title>DAVID VARGAS PUGA</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div class="all-content" id="content">
-        <table id="allTable" class="table-allInfo">
+        <table>
             <tr>
-                <td id="name">David Vargas Puga</td>
-            </tr>
-            <tr>
-                <td id="aboutMe">About Me</td>
+                <td>
+                    <img id="avatar" class="circular-square" src="./images/avi.jpg">
+                </td>
             </tr>
             <tr>
-                <td bgcolor="#FFA500" id="aboutMe-info">A bilingual and extroverted Computer Science and Spanish double-major with exceptional communication, interpersonal and leadership skills. First-generation student who thrives in cultures that embrace diversity and reward imagination. Experience in software engineering, Android development and machine learning. </td>
+                <td id="name"><h1>David Vargas Puga</h1></td>
```

Comment:
name is still unused, please delete all unnecessary ids and class names

Reply:
hey marcel, i do use that id in my css file.